vme/src/vmc/vmc_process.cpp

```diff
@@ -1016,12 +1016,6 @@ void process_unit(unit_data *u)
     }
     u->setWeight(UNIT_BASE_WEIGHT(u));
 
-    if (!is_in(UNIT_ALIGNMENT(u), -1000, +1000))
```

Comment:
I was wondering if it wouldn't be nicer to have a "bool isAlignmentInRange(value)" so that we don't have an ugly #ifdef over in the alignement code & then we localize special code in vmc code rather than in generic code.

I.e. keep these line 1029 .. 1024 but either do is_in(Min, MaxAlignment) or do the isAlignmentInRange()